install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -57,19 +61,6 @@ data:
             - NET_ADMIN
           privileged: true
         restartPolicy: Always
-      {{ if eq .Values.global.proxy.enableCoreDump true }}
-      - args:
-        - -c
-        - sysctl -w kernel.core_pattern=/etc/istio/proxy/core.%e.%p.%t && ulimit -c unlimited
-        command:
-          - /bin/sh
-        image: {{ .Values.global.hub }}/proxy_init:{{ .Values.global.tag }}
-        imagePullPolicy: IfNotPresent
-        name: enable-core-dump
-        resources: {}
-        securityContext:
-          privileged: true
-      {{ end }}
```

Comment:
This seems to have been totally off... I hope I'm reading it right because it's a major change.

Reply:
This is adding a new init container to only run `sysctl`

Also change it to
`{{ if .Values.global.proxy.enableCoreDump }} `
`eq true` is not needed.